content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-framework.md

```diff
@@ -88,7 +88,7 @@ Install the Datadog .NET Tracer machine-wide so that all services on the machine
 
 To install the .NET Tracer machine-wide:
 
-1. Download the [.NET Tracer MSI installer][1]. Select the MSI installer for the architecture that matches the operating system (x64 or x86).
+1. Download the [.NET Tracer MSI installer][1].
```

Comment:
Should we note that they should be using the x64 installer?

Reply:
Discussed above - I'll update this too if you agree with the suggestion